src/nerve/frontends/tui/commander/executor.py

```
@@ -462,3 +467,158 @@ async def execute_python_command(
+        # Send answer to workflow
+        result = await adapter.answer_gate(run_id, answer)
+        if not result.get("success"):
+            console.print(f"[error]Failed to answer gate: {result.get('error')}[/]")
+
+    except (KeyboardInterrupt, EOFError):
+        console.print("[dim]Gate input cancelled[/]")
+        # Try to cancel the workflow
+        try:
+            await adapter.cancel_workflow(run_id)
+        except Exception:
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)